app/constants.js

```diff
@@ -1,10 +1,12 @@
 const path = require('path')
 
-const { app } = require('electron').remote
+const app = require('electron').app
+const remote = require('electron').remote
+const api = app || remote.app
```

Comment:
@greggraf I think the simplest linting add would be `npm install --save-dev standard`, then `"test": "standard"` in the scripts section of `package.json`. Eslint is overkill for now probably.

Unit testing should wait until we have the units figured out - which is essentially this series of PRs plus some modularisation. My preference would be to add tests to each module as we break it out into a separate repo.